finality-aleph/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "finality-aleph"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
I think we avoid bumping this version in most PRs, so please do so here as well, for consistency.